cilium/ipcache.cc

```diff
@@ -28,11 +28,11 @@ namespace Cilium {
 // them to a separate include file we can include here instead of
 // copying them!
 
-typedef uint32_t __be32; // Beware of the byte order!
-typedef uint64_t __u64;
-typedef uint32_t __u32;
-typedef uint16_t __u16;
-typedef uint8_t __u8;
+using __be32 = uint32_t; // Beware of the byte order!
+using __u64 = uint64_t;
+using __u32 = uint32_t;
+using __u16 = uint16_t;
+using __u8 = uint8_t;
 
 PACKED_STRUCT(struct ipcache_key {
```

Comment:
rename struct to `IPCacheKey`? (getting `Invalid case style for struct 'ipcache_key' (fix available)`)

Reply:
Are you getting this from `clangd` or from running `make tidy`?

Applied.

Reply:
`clangd` - but i assume that it now respects the clang-tidy config too

Reply:
Renamed as "IpCacheKey"